### PR TITLE
Portfolio Desktop version

### PR DESCRIPTION
**Pull request by Sofinaye**
I added :
Responsiveness for wider screen sizes.
Expandable navbars.
and make changes in the layout.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -173,11 +184,13 @@ <h3>Professional art <span>printing Data</span></p>
         <button type="" class="project1-btn">See project</button>
```
Comment (ha-manel) [inline index.html:184]: - [x] Great job with adding the hover effect and the animation on hover 👏 However, kindly use the styling shown in Figma👍 
- [x] Also please add the hover effect on the button in the contact section too 💯 

image

File: style.css [hunk 2]
```diff
@@ -454,3 +467,255 @@ html {
 .footer-social-item {
   color: #505f79;
 }
+
+.header-overly {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
```
Comment (ha-manel) [inline style.css:484]: - [x] Good job on making both the desktop and mobile versions look great 👏 However there are still some issues with some screen sizes, the website is not fully responsive yet, kindly check the gif below and try to fix the issues 👍 

Animation

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi Team👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

## Optional:
- [ ] Please consider using best practices when you write commit messages, they should be capitalized sentences in the imperative form, example: "Add linters" instead of "added linters" or "linters are added" ... etc 👍 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @Sofinaye, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**